src/utils.py

```diff
@@ -2,6 +2,8 @@
 import json
```

Comment:
Nitpick discussion: what do you think about moving the class `class DuneVersion(Enum)` to this file as well?

Reply:
Actually, its a pretty tiny class so I will just go ahead and move it now.